x-pack/solutions/search/plugins/search_synonyms/public/plugin.ts

```diff
@@ -30,7 +31,7 @@ export class SearchSynonymsPlugin
     }
     core.application.register({
       id: PLUGIN_ID,
-      appRoute: '/app/elasticsearch/synonyms',
+      appRoute: PLUGIN_ROUTE_ROOT,
       title: PLUGIN_TITLE,
       deepLinks: [
```

Comment:
we should consider if we want a deeplink or locator for the synonym set detail page, not need now but something to keep in mind.

Reply:
I can try removing those.